packages/components/src/components/Timerange/index.style.tsx

```diff
@@ -165,7 +165,7 @@ export const style = css`
         display: flex;
         align-items: center;
         justify-content: center;
-        z-index: 2;
+        z-index: 1;
```

Comment:
![Screenshot 2022-05-18 at 9 28 31](https://user-images.githubusercontent.com/33235762/168982227-3e1589de-0de4-47ac-8652-71bed021c1f5.png)
![Screenshot 2022-05-18 at 9 28 53](https://user-images.githubusercontent.com/33235762/168982304-1e0d34d6-4d71-44aa-8d12-eb9abcebf888.png)

Not a Timerange but probably the same problem.
![Screenshot 2022-05-18 at 9 29 12](https://user-images.githubusercontent.com/33235762/168982374-f6391565-d2dc-42c5-84b8-9572e2b3df13.png)